packages/maskbook/src/social-network-adaptor/twitter.com/injection/EnhancedProfileTab.tsx

```diff
@@ -58,7 +58,7 @@ const useStyles = makeStyles<StyleProps>()((theme, props) => ({
         fontWeight: 700,
     },
     line: {
-        dispaly: 'inline-flex',
```

Comment:
Break change. 😂

Reply:
🤣 Maybe we should remove this line since it doesn't affect the layout before

Reply:
Make sense. I have remove it.